backend/typescript/models/testSession.model.ts

```diff
@@ -0,0 +1,83 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+/**
+ * This interface holds information about the result of a single student
+ * on a test
+ */
+export interface Result {
+  /** the name of the student */
+  student: string;
+  /** the score of the student */
+  score: number;
+  /**
+   * a list corresponding to the question list with each field indicating
+   * the student's answer
+   */
+  breakdown: [number];
```

Comment:
Sorry, I just realized we need both `breakdown` and `answers`. i.e we should store what the student wrote and whether or not the answer is correct.